pysindy/pysindy.py

```diff
@@ -8,6 +8,7 @@
 from numpy import vstack
 from numpy import zeros
 from scipy.integrate import odeint
+from scipy.integrate import solve_ivp
```

Comment:
What's the motivation behind adding `solve_ivp`?

Reply:
seems to be the standard for python ODEs these days. Originally, I hoped it could solve differential algebraic equations (DAEs) of the sort that SINDy-PI produces. It appears not... need special functionality (see the example notebook). My recommendation would be to change the main branch to use solve_ivp for everything.